en/components/accordion.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Accordion - Design System Components
+_description: An accordion is a user interface component that allows the user to hide or reveal content.
+_keywords: Design Systems, Design Systems UX, UI kit, Figma, Figma to Angular, Export code from Figma, Figma to HTML, Figma UI kits, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits, Adobe XD, Adobe XD to Angular, Export code from Adobe XD, Adobe XD to HTML, Adobe XD UI kits
+---
+
+## Accordion
+
+The Accordion component allows users to reveal content in a vertical stack progressively. It typically consists of a list of [Expansion Panels](expansion-panel.md), where each item can be expanded or collapsed to reveal or hide its content. Accordions are often used to manage a large amount of information in a compact space and to allow users to focus on specific pieces of information selectively. The Accordion is visually identical to the [Ignite UI for Angular Accordion Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/accordion)
+
+## Accordion Demo
+
+<img class="responsive-img" src="../images/accordion_demo.png" srcset="../images/accordion_demo@2x.png 2x" />
+
+## Using the Accordion in Figma and Sketch
+
+The Accordion comes with 5 panels. Each panel consists of top and bottom margins, which can be turned on/off with the boolean operations. By going into the nested Expansiоn Panels, you can modify their contents. If you need to design an Accordion with more Expansion Panels, the easiest way to do it is by detaching the component. To detach the component, you need to place it in your workspace (Frame or Artboard), right-clicking on it will open a contextual menu where `Detach Instance` (`Detach from Symbol` in Sketch) should be selected.
+
+After detaching the component, you will be able to add your additional Expansion Panels by duplicating any of the existing ones. In Figma, the size will be adjusted automatically thanks to the Auto Layout. In Sketch, you should manually adjust the additionally added Expansion Panels' position.
+
+## Using the Accordion in Adobe XD
+
+> [!NOTE]
+> WIP: Currently, we don't support the Accordion component in Adobe XD, but you can also combine multiple `Expansion Panels` into a similar Accordion component behavior.
```

Comment:
Maybe we should also say something like "However, this will not generate appropriately in the App Builder" or something similar.

Reply:
As far as I see, it is generated the same way as the components from the library.

Reply:
I'm talking about the Accordion in Adobe XD. It will probably generate separate expansion panels instead of an accordion due to the lack of accordion metadata.

Reply:
Fixed